client/components/redux-forms/redux-form-radio/index.jsx

```diff
@@ -1,43 +1,24 @@
+/** @format */
 /**
  * External dependencies
  */
-import React, { Component, PropTypes } from 'react';
+import React from 'react';
+import PropTypes from 'prop-types';
 import { Field } from 'redux-form';
-import { omit } from 'lodash';
 
 /**
  * Internal dependencies
  */
 import FormRadio from 'components/forms/form-radio';
 
-class ReduxFormRadio extends Component {
-	static propTypes = {
-		name: PropTypes.string,
-		value: PropTypes.string,
-	};
+// eslint-disable-next-line no-unused-vars
+const RenderRadio = ( { input, meta, type, ...props } ) => <FormRadio { ...input } { ...props } />;
```

Comment:
Also, should we continue to destructure `input` so it's more explicit?

```js
const RenderRadio = ( { input: { name, onChange, value }, meta, type, ...props } ) => <FormRadio name={ name } onChange={ onChange } value={ value } { ...props } />;
```

This is slightly more verbose, but it would document what values we're expecting. (Tho it might be fine to stick with `{ ...input }` if that value is always provided by `redux-forms` anyway, which we can assume to do the right thing.)